main.js

```diff
@@ -85,6 +85,10 @@ app.on('ready',() => {
         switch (json.name) {
             // App
             case consts.eventNames.appCmdQuit:
+            rl.close()
+            setTimeout(function(){
```

Comment:
Why do you add a timeout here? And won't it be cancelled by `app.quit()` just  below?

Reply:
I was testing something around this, might have missed removing this code.

Reply:
removed code